projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -97,6 +103,29 @@ function jetpack_mark_if_post_answers_blogging_prompt( $post_id, $post, $update,
  * Utility functions.
  */
 
+/**
+ * Retrieve a blogging prompt by prompt ID.
+ *
+ * @param int $prompt_id ID of the prompt fetch.
+ * @return stdClass|null Prompt object or null.
+ */
+function jetpack_get_blogging_prompt_by_id( $prompt_id ) {
+	$locale = get_locale();
+	$route  = sprintf( '/wpcom/v3/blogging-prompts/%d', $prompt_id );
+
+	$request = new WP_REST_Request( Requests::GET, $route );
```

Comment:
I believe `Requests` will be deprecated starting in WordPress 6.2:

https://make.wordpress.org/core/2023/03/08/requests-library-upgraded-to-2-0-5-in-wordpress-6-2/

Since we need to support WordPress 6.1 and WP 6.2, we'll need to handle both here I'm afraid.

Reply:
Thanks for flagging this!

Correct me if I'm wrong, but I don't think those changes affect the [WP_REST_Request class](https://github.com/wordpress/wordpress-develop/blob/6.2/src/wp-includes/rest-api/class-wp-rest-request.php), only the underlying library for making external http requests.

Reply:
oh, the `Requests::GET` is the problem...I'll fix that.